src/__tests__/profiling-test.js

```diff
@@ -16,17 +16,19 @@ describe('profiling', () => {
 
   const exportImportHelper = (rendererID: number, rootID: number) => {
     const {
-      prepareProfilingExport,
-      prepareProfilingImport,
+      prepareExportedProfilingSummary,
+      prepareImportedProfilingData,
     } = require('src/devtools/views/Profiler/utils');
 
-    let exportedProfilingSummary;
+    let exportedProfilingDataJsonString = '';
     bridge.addListener('exportFile', ({ contents }) => {
-      exportedProfilingSummary = contents;
+      if (typeof contents === 'string') {
+        exportedProfilingDataJsonString = (contents: string);
+      }
     });
 
     utils.act(() => {
-      const exportProfilingSummary = prepareProfilingExport(
+      const exportProfilingSummary = prepareExportedProfilingSummary(
```

Comment:
Note that there are two separate types of "exported": 
- `ExportedProfilingSummaryFromFrontend` that gets created by `prepareExportedProfilingSummary` 
  (⚠️ _previously this function was named as `prepareProfilingExport` but this name was confusing because it's not an exact mirror of `prepareProfilingImport`_ ⚠️), and 
- `ExportedProfilingData` that gets built by the `exportProfilingSummary` handler in the `agent` (created by `prepareExportedProfilingData`).

And one type of "imported":
- `ImportedProfilingData`.